moveit_experimental/jog_arm/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+add_executable(jog_arm_server 
```

Comment:
use a variable to define "jog_arm" and use as a namespace in all executables in this CMakeList, including spacenav_*

see https://github.com/ros-planning/moveit/blob/melodic-devel/moveit_core/robot_state/CMakeLists.txt#L1